Dockerfile

```diff
@@ -4,7 +4,7 @@ RUN dnf module -y install python39 && dnf install -y python39 python39-pip && dn
 
 RUN mkdir /stressng
 RUN chmod 777 /stressng
-ADD https://raw.githubusercontent.com/arcalot/arcaflow-plugins/main/LICENSE /stressng/
+ADD https://raw.githubusercontent.com/arcalot/arcaflow-plugin-template-python/main/LICENSE /stressng/
```

Comment:
Are we sure this is what we want here?  Don't we want to use the LICENSE of this plugin?

Reply:
I think the issue here is that carpenter requires this line to be here.
I would consider just checking if the license included in the plugin differs from the one in the template instead of forcefully consume the license shipped within the template.

Reply:
@portante and I were talking about this on another pr....I will copy paste my answer. Def should get some clarity on what we want to do here. Seems to make the most sense to get the LICENSE file(s) from the current dir.

Y`eah that license question for the PR's is one that I was trying to ask last week. If we ant to incorporate the LICENSE of that repo we will need to change carpenter's format for checking for the file. This is because carpenter is hard coded to look for "(ADD|COPY) .*/LICENSE /.*": ....so doing an COPY LICENSE /app/ will cause it to fail
[10:08](https://coreos.slack.com/archives/D0495JT2QCE/p1668096501432879)
easy fix but I didn't get a straight forward answer what to do besides use the template LICENSE file as the old one is deprecated.`